src/parsers/installrdf.js

```diff
@@ -105,6 +109,40 @@ export default class InstallRdfParser {
       this._getTopLevelNodeByTag('em:bootstrap')) === 'true';
   }
 
+  _getApplication() {
+    var targetApplications = this._getTopLevelNodesByTag(
+      'em:targetApplication');
+    var applications = [];
+
+    for (let targetApplication of targetApplications) {
+      let application = {};
+      let descriptionNodes = Array.prototype.slice.call(
+          targetApplication.childNodes)
+          .filter((node) => node.nodeName === 'Description');
+
+      if (descriptionNodes.length !== 1) {
+        throw new Error(singleLineString`<em:targetApplication> node should only
+          have a single descendant <Description>`);
+      }
+
+      for (var i in descriptionNodes[0].childNodes) {
+        let childNode = descriptionNodes[0].childNodes[i];
+        if (TARGET_APPLICATION_FIELDS.indexOf(childNode.localName) !== -1) {
```

Comment:
Should be able to use `Array.prototype.includes` [1] here.

[1] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes